org.js.nuclear.Nuclear.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "org.js.nuclear.Nuclear",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "branch": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "separate-locales": "false",
+    "command": "run.sh",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host",
+	"--socket=pulseaudio"
+    ],
+    "build-options" : {
+        "env": {
+            "NPM_CONFIG_LOGLEVEL": "info"
+        }
+    },
+    "modules": [
+        {
+            "name": "nuclear",
+            "buildsystem": "simple",
+            "sources": [
+               {
+                   "type": "archive",
+                   "url": "https://github.com/nukeop/nuclear/releases/download/v0.6.3/nuclear-fca030.tar.gz",
+                   "sha256": "5b90a1064736c158f62ea74c5814dffdd25f8b6a3877dfeed065377e744cdd2e"
+                },
+                {
+                    "type":"file",
+                    "url":"https://raw.githubusercontent.com/advaithm/nuclear/master/flatpak/org.js.nuclear.Nuclear.metainfo.xml",
+                    "sha256":"17f2d8221f3cfec579457c05a4a56b85dcff2dd44bc86d5a2bca85e088af7c39",
+                    "dest-filename":"org.js.nuclear.Nuclear.metainfo.xml"
+                },
+                {
+                  "type":"file",
+                  "url":"https://raw.githubusercontent.com/advaithm/nuclear/master/flatpak/org.js.nuclear.Nuclear.desktop",
```

Comment:
if you are pulling directly from git like that, please do it on a specific commit. `master` will move and the content might change.

(same above and below)

Reply:
We are going to be making a "mirror" repo which won't change unless the flatpak does too

Reply:
Either these file are in this repository (it's often the case for .desktop or .appdata.xml that aren't yet upstream) or you refer to them with a specific commit or tag.

Reply:
is it possible to continue using the icons from the main repo